.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+
+# Add a reference to related issue
+Fixes: #NUMBER
```

Comment:
Preferably into the git commit message, so people in command line have the info, too.

Reply:
updated.